controllers/drplacementcontrol_controller.go

```diff
@@ -1480,13 +1590,43 @@ func (d *DRPCInstance) getVRGFromManifestWork(mw *ocmworkv1.ManifestWork) (*rmn.
 func (d *DRPCInstance) updateDRPCStatus() error {
 	d.log.Info("Updating DRPC status")
 
-	d.instance.Status.LastUpdateTime = metav1.Now()
+	drpc := &rmn.DRPlacementControl{}
+	drpcLookupKey := types.NamespacedName{
+		Name:      d.instance.Name,
+		Namespace: d.instance.Namespace,
+	}
+
+	err := d.reconciler.Client.Get(d.ctx, drpcLookupKey, drpc)
+	if err != nil {
+		d.log.Error(err, "failed to update status1")
+
+		return fmt.Errorf("%w", err)
+	}
+
+	drpc.Status = d.instance.Status
 
-	if err := d.reconciler.Status().Update(d.ctx, d.instance); err != nil {
+	if len(d.userPlacementRule.Status.Decisions) != 0 {
```

Comment:
(future) we can also reflect status of Secondary VRG here in the future, so that cleaning status etc. can be viewed by the user